src/firefox/index.js

```diff
@@ -239,8 +244,32 @@ export type UseProfileParams = {
   app?: PreferencesAppName,
   configureThisProfile?: ConfigureProfileFn,
   customPrefs?: FirefoxPreferences,
+  createProfileFinder?: typeof defaultCreateProfileFinder,
 };
 
+export function defaultCreateProfileFinder() {
```

Comment:
All the new tests are using a fake `createProfileFinder`, and so (like the code coverage reports confirm) it is time to create some tests specifically for defaultCreateProfileFinder).

To make this function more easily testable we should add an optional `userDirectoryPath` (a string) parameter to it, so that we can customize the directory where the `Finder` class will search the `profiles.ini` file and the profile directories.